flight/Modules/Actuator/actuator.c

```diff
@@ -83,13 +86,18 @@ static bool manualControlCommandUpdated = true;
 // set true to ensure they're fetched on first run
 static volatile bool actuator_settings_updated = true;
 static volatile bool mixer_settings_updated = true;
```

Comment:
We may want to make all of this stuff conditional on not-F1, because RAM is tight.